srcpkgs/mpvpaper/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'mpvpaper'
+pkgname=mpvpaper
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="mpv-devel wlroots-devel"
+depends="mpv wlroots"
```

Comment:
It links against `libmpv`, so `mpv` in `depends` isn't needed... But does it actually touch `wlroots`?

Reply:
Yeah right `wlroots` and `mpv` don't seem to be explicitly needed in depends.

But it uses xdg-output-unstable-v1 and wlr-layer-shell-unstable-v1 so shouldn't we specify `wlroots` in depends as a safe guard (since it only works in wlroots based compositors)?

Reply:
I don't think so. You can implement wlr protocols without being wlroots based, even.

Reply:
I understand the need for `wlroots-devel`, then.